docs/stable/2-account.md

```diff
@@ -213,12 +220,193 @@ All messages sent are encrypted with the public key of the destination and signe
 
 -->
 
+## Public Key Compression
+
+The node MUST provide functionality for the compression and decompression of public / chat keys.
```

Comment:
SHOULD? Otherwise previous versions would be incompatible with the account spec, which seems off.

Reply:
Yes, you make a very good point.